wiki/FAQ/en.md

```diff
@@ -56,161 +78,258 @@ _[Osu! Tatakae! Ouendan][ouendan wikipedia]_ is a rhythm game for the Nintendo D
 
 ![Gameplay footage of Osu! Tatakae! Ouendan in Nintendo DS][ouendan image]
 
-You can see the circles on the bottom screen, and on the top screen you have the story. Each stage is basically a self-contained story about a person in trouble. That's where the _Ouendan_ (Cheer squad) comes in. Through the magical power of male cheer-leading, you have to help the people in trouble.
+_Gameplay footage of Osu! Tatakae! Ouendan in Nintendo DS_
 
-### How do I play?
-_Full explanation: [osu!][osu wikilink]_
+You can see the circles on the bottom screen, and on the top screen you have the story. Each stage is basically a self-contained story about a person in trouble. That is where the _Ouendan_ (Cheer squad) comes in. Through the magical power of male cheerleading, you have to help the people in trouble.
 
-While most PC users won't have a touchscreen at their disposal, this game is made to be playable with a normal mouse, as well as any number of input devices you can possibly think of. Different play styles can be found in the [Play Styles][Play_Styles wikilink] page. There are some songs which will push the limits of the normal mouse (and you), but all the (ranked and approved) beatmaps have been tested and (most can be) passable using only a mouse.
+### How to play?
+_Full explanation: [osu!][osu! wikilink]_
 
-#### Game-play Basics
-_Tutorial: [(Bundled with PC osu!) Original tutorial][osu!tutorial_basic], [Ranked beatmap version][osu!tutorial_rank]_
+While most players may not have a touchscreen at their disposal, this game was made to be playable with a normal mouse in mind, as well as any creative variations of input devices usage. Different play styles can be found in the [Play Styles][Play_Styles wikilink] page. There are some songs which will push the limits of the normal mouse and the player, but all the (ranked and approved) beatmaps have been tested and (most can be) passable using only a mouse.
```

Comment:
The `Play Styles` link should have its text lowercased. There isn't anything proper about it.